man/bootctl.xml

```diff
@@ -256,6 +256,13 @@
         <xi:include href="version-info.xml" xpointer="v243"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>set-log-level</option></term>
```

Comment:
For `systemctl`, we initially had `set-log-level` to set and `log-level` to get. But then we realized that just one verb is enough and the correct behaviour can be inferred from the presence or lack of the argument. Please just add `log-level` here.